RealmTasks Apple/RealmTasks iOS/TableViewCell.swift

```diff
@@ -73,22 +73,22 @@ final class TableViewCell<Item: Object where Item: CellPresentable>: UITableView
     var presenter: CellPresenter<Item>!
 
     // Private Properties
-    private var originalDoneIconCenter = CGPoint()
-    private var originalDeleteIconCenter = CGPoint()
+    fileprivate var originalDoneIconCenter = CGPoint()
```

Comment:
These properties are not used in any extensions, so the could still be `private`